lambda_code/tests/test_immunization_pre_validator.py

```diff
@@ -131,7 +131,6 @@ def test_pre_validate_patient_identifier_value(self):
             self,
             field_location="contained[?(@.resourceType=='Patient')].identifier[0].value",
             valid_strings_to_test=["9990548609"],
-            defined_length=10,
```

Comment:
Why remove this? This requirement remains (the field should not exist at all if the value is None)

Reply:
added back in